src/cli/Utils.cpp

```diff
@@ -135,6 +136,38 @@ namespace Utils
             compositeKey->addKey(fileKey);
         }
 
+#ifdef WITH_XC_YUBIKEY
+        if (!yubiKeySlot.isEmpty()) {
+            bool ok = false;
+            int slot = yubiKeySlot.toInt(&ok, 10);
+            if (!ok) {
+                err << QString("Invalid YubiKey slot %1").arg(yubiKeySlot) << endl;
+                return {};
+            }
+            if (slot != 1 && slot != 2) {
+                err << QString("Invalid YubiKey slot %1").arg(yubiKeySlot) << endl;
+                return {};
+            }
+
+            QString errorMessage;
+            bool blocking = YubiKey::instance()->isBlocking(slot, errorMessage);
+            if (!errorMessage.isEmpty()) {
+                err << errorMessage << endl;
+                return {};
+            }
+
+            auto key = QSharedPointer<YkChallengeResponseKeyCLI>(new YkChallengeResponseKeyCLI(
+                slot,
+                blocking,
+                QString("Please touch the button on your YubiKey to unlock " + databaseFilename),
```

Comment:
This message should be embedded in the key instance, no need to repeat the filename.

Reply:
The filename is useful for example if you merge 2 database files from the CLI. It makes it clear which database the interaction is related to. I can remove the filename and let the `YkChallengeResponseKeyCLI` handle the interaction in a generic way tho, your call.